Drop-In/src/androidTest/java/com/braintreepayments/api/SelectPaymentMethodParentFragmentUITest.kt

```diff
@@ -40,6 +40,7 @@ class SelectPaymentMethodParentFragmentUITest {
 
     @Test
     fun whenStateIsRESUMED_onShowVaultManagerEvent_showsVaultManager() {
+        // TODO: determine how to test UI with animation delay
```

Comment:
This test is not in the Demo app integration tests so not specifically covered in this PR, but it's the only one still failing - I think due to the animation changes. The functionality is working as expected but the test fails because the assertion doesn't wait for the animation to complete. I couldn't find a good way to test with animation delay here. We could disable animation on the simulator before running, but I'd rather not add another manual step to the testing process. Maybe we should remove this test and test the functionality through a Demo app integration test?

Reply:
We could I ran across this issue in the refactor loader branch though. I put in a few wait for 1-second statements to get them all to pass.

Reply:
Ah good call - added a wait to this test 👍